src/dataflow-types/src/client/replicated.rs

```diff
@@ -25,18 +25,77 @@
 use std::collections::{HashMap, HashSet};
 
 use timely::progress::{frontier::MutableAntichain, Antichain};
+use tokio::sync::mpsc::{unbounded_channel, UnboundedReceiver, UnboundedSender};
+use tokio::task::JoinHandle;
 
 use crate::client::Peek;
 use mz_repr::GlobalId;
 
 use super::{ComputeClient, GenericClient};
 use super::{ComputeCommand, ComputeResponse};
 
+// `Client` must have a cancelation-safe `recv` method.
+pub fn spawn_client_tasklet<
```

Comment:
I think the term you're looking for is just "task", no?

```suggestion
pub fn spawn_client_task<
```